src/core/db/sqlite.go

```diff
@@ -226,6 +238,26 @@ func (db *SQLite) createTables(ctx context.Context) error {
 	}
 	return nil
 }
+func (db *SQLite) createSnapshotServiceTables(ctx context.Context) error {
```

Comment:
do you actually re-use any of the database functions? why are we even trying to share database functions? The only thing we care to share is the indexer. the database writing code isn't changing based on a 3rd party